### PR TITLE
Update icons & wording to make them specific to NYS

### PR DESCRIPTION
**Pull request by mscharfstein**
nyforme

### CODE REVIEW

File: client/common/locales/en/translation.json [hunk 1]
```diff
@@ -44,23 +44,23 @@
     "applyButton": "Apply"
   },
   "serviceList": {
-    "pageTitle": "Services for Me",
-    "title": "Services for me",
+    "pageTitle": "New York for Me",
```
Comment (kebrahim) [inline client/common/locales/en/translation.json:47]: how about "New York State Services" to match https://www.ny.gov/services?

File: client/common/locales/en/translation.json [hunk 2]
```diff
@@ -44,23 +44,23 @@
...
+    "title": "New York for me",
```
Comment (kebrahim) [inline client/common/locales/en/translation.json:48]: "New York State Services"

File: client/common/locales/es/translation.json [hunk 3]
```diff
@@ -30,24 +30,24 @@
...
     "unsureLinkText": "No estoy seguro, ayúdame a comprobar"
   },
   "filterModal": {
     "clearButton": "Borrar",
     "applyButton": "Aplicar"
   },
   "serviceList": {
-    "pageTitle": "Servicios para mí",
-    "title": "Servicios para mí",
+    "pageTitle": "Nueva York para mí",
```
Comment (kebrahim) [inline client/common/locales/es/translation.json:46]: if you change the above, i suppose you'd also have to change these :)

File: client/common/locales/es/translation.json [hunk 4]
```diff
@@ -30,24 +30,24 @@
...
+    "title": "Nueva York para mí",
```
Comment (kebrahim) [inline client/common/locales/es/translation.json:47]: and this one

**Review by kebrahim — APPROVED** [review]
all comments are optional. only change them if you agree it makes sense. otherwise, i'm fine w/ what you have

**Comment by mscharfstein** [comment]
I kinda am biased to leaving it how it is to show that it's actually different from the existing Services pathway. Doesn't really matter since we won't actually be using this in the long run :)

**Comment by kebrahim** [comment]
[Duplicate component to view different estimations on FE](https://trello.com/c/9MnbRq1D/3-duplicate-component-to-view-different-estimations-on-fe)

**Comment by kebrahim** [comment]
[Rebrand Boulder for Me for NYS](https://trello.com/c/3dhoK2jr/4-rebrand-boulder-for-me-for-nys)

**Pull request merged**